wpengine/faustjs
# Auto Version

## Tasks


## Description

These changes add automatic versioning to the plugin's enqueued assets.

## Testing

1. Checkout this branch: `MERL-1131-auto-version`
2. Visit the plugin settings page: `/wp-admin/options-general.php?page=faustwp-settings`
3. Open your browser's devtools & search for `#faustwp-settings-css`
4. Observe that the current version of the plugin matches the query arg
5. Visit the block editor (visit any page or post)
6. Open your browser's devtools & search for `#faustwp-gutenberg-filters-js`
7. Observe that the current version of the plugin matches the query arg

## Screenshots

<img width="1621" alt="Screenshot 2023-08-02 at 10 00 51 AM" src="https://github.com/wpengine/faustjs/assets/6676674/8e627722-3da3-4232-ba9d-4b2db9aabed2">
<img width="1608" alt="Screenshot 2023-08-02 at 10 08 45 AM" src="https://github.com/wpengine/faustjs/assets/6676674/41f20b1b-fbdb-4f4e-990b-fa44d508fab9">
